bedrock/firefox/templates/firefox/tiles.html

```diff
@@ -45,197 +46,99 @@
-We believe that these are beneficial for every Web user to know about to help advance our mission to promote openness, innovation and opportunity on the Web.
-</p>
-
-<h3>Easy to control</h3>
-
-<p>
-Tiles are easily pinned, moved around or removed using simple drag-and-drop and close interfaces.
-If you do not want to see any Tiles, you can deactivate them completely in two clicks through the new tab gear control.
-</p>
-
+  <h2>{{ _('If you\'re getting started') }}</h2>
+  <p>{{ _('If you\'re using Firefox for the first time, the New Tab page is already populated with useful and interesting sites and content. This includes both referrals to sponsored sites from commercial partners, labeled as "Sponsored," and content we consider important to our mission of advancing the public benefit of the Internet.') }}</p>
+  <img src="{{ static('img/firefox/tiles/discover-sites.png') }}" alt="{{ _('Screenshot') }}">
+{% else %}
+  <h2>{{ _('What are Directory Tiles?') }}</h2>
+  <p>
+  {{ _('New Firefox users are initially given a set of Directory Tiles as suggestions for sites that may be of interest to them.') }}
+  {{ _('Instead of displaying a blank set of Tiles, Firefox shows content from Mozilla and some Mozilla partners.') }}
+  {{ _('All partner content is marked as “sponsored” and users have complete control to edit or remove any Tile.') }}
+  </p>
 {% endif %}
+</section>
 
-<h3>{{ _('Respects your privacy') }}</h3>
-
-<p>
+<section class="tls-privacy">
 {% if LANG in ['en-US'] %}
-Tiles are a part of our initiative to set new standards for Internet advertisers and content publishers in the collection and use of personal data.
-</p>
-<p>
-Mozilla respects your privacy and any data collection by Mozilla via Tiles will be done in accordance with our <a href="{{ url('privacy.notices.firefox') }}">Privacy Notice</a>.
-The data we collect is to help us deliver the most interesting and relevant content to our users.
-For more on what data Mozilla collects via Tiles, see below.
+  <h2>{{ _('Personalization with your privacy in mind') }}</h2>
+  <p>{{ _('Mozilla promotes a transparent exchange between users and content providers to make the Web enjoyable and sustainable.') }}</p>
+  <p>{{ _('If suggestions are enabled, Firefox will make suggestions to you based on categories identified in your browsing history and will let you know every time why are you seeing specific suggestions.') }}</p>
+  <p>{{ _('In delivering Suggested sites, Firefox does not share your personal data, nor place cookies on your computer. Content providers will only receive aggregated information about interactions users have with their tiles, but individual information is never shared or sold.') }}</p>
+  <p>{{ _('You remain in control of your browsing history, and you can disable suggestions whenever your chose.') }}</p>
```

Comment:
I know it is like this in the copy but, I am pretty sure "disable suggestions whenever your chose." should be "disable suggestions whenever *you choose*."